.github/workflows/tls_codec.yml

```diff
@@ -70,4 +70,4 @@ jobs:
       - run: ${{ matrix.deps }}
```

Comment:
Looks like there's a typo in the matrix "targets" -> "target".

Reply:
Since this PR is merged, I'll handle the typo as part of another PR.

Reply:
Yes, for some reason this works even with this typo. I'm not entirely sure why. But the linter complained.